pkg/services/alerting/engine.go

```diff
@@ -17,44 +17,43 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// AlertingService is the background process that
+// schedules alert evaluations and makes sure notifications
+// are sent.
```

Comment:
Super nitpicking here: I usually either use dots everywhere or nowhere when commenting APIs

Reply:
Good point. I dont think we have a decision for this. But comments are supposed to be presented as docs. as such I think dots makes sense since it should be read by human.